tools/c7n_azure/c7n_azure/resources/advisor.py

```diff
@@ -0,0 +1,45 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('advisor-recommendations')
```

Comment:
minor: for other resources we use single form, so I'd suggest renaming it to `advisor-recommendation`